kaltura/playkit-js-smartswitch
# check full flow

- test tests
- test tests
- test canary full flow
- prepare gitActions
- revert to master
- disable travis execution
- test notification
- test notification
- fix workflows
- test canary
- test canary
- test prod
- test prod
- test prod
- test prod
- chore(release): 0.2.1
- reset to master
- reset to master
- update version
- chore(release): 0.2.2
- chore(release): 0.2.3
- revert
- check full flow

### Description of the Changes

Please add a detailed description of the change, weather it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### CheckLists
